metagenomics_samplesheet_generator.ipynb

```diff
@@ -2,15 +2,44 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "metapool    : 2021.11+0.gcec558b.dirty\n",
+      "sample_sheet: 0.12.0\n",
+      "openpyxl    : 3.0.7\n",
+      "\n",
+      "Compiler    : Clang 4.0.1 (tags/RELEASE_401/final)\n",
+      "OS          : Darwin\n",
+      "Release     : 20.6.0\n",
+      "Machine     : x86_64\n",
+      "Processor   : i386\n",
+      "CPU cores   : 8\n",
+      "Architecture: 64bit\n",
+      "\n",
+      "Hostname: debugatron-4000.attlocal.net\n",
+      "\n",
+      "matplotlib: 3.2.0\n",
+      "pandas    : 1.0.2\n",
+      "seaborn   : 0.10.0\n",
+      "re        : 2.2.1\n",
+      "numpy     : 1.18.1\n",
+      "\n"
+     ]
+    }
+   ],
    "source": [
+    "%reload_ext watermark\n",
     "%matplotlib inline\n",
     "\n",
     "import os\n",
     "from metapool.metapool import *\n",
-    "from metapool import validate_sample_sheet, make_sample_sheet, IGMManifest"
+    "from metapool import validate_and_scrub_sample_sheet, make_sample_sheet, IGMManifest\n",
+    "%watermark -iv -m -h -p metapool,sample_sheet,openpyxl"
```

Comment:
could this also include the version of python?

Reply:
Sure thing, added!